packages/fast-usdc/src/exos/transaction-feed.js

```diff
@@ -66,6 +68,10 @@ export const stateShape = {
 };
 
 /**
+ * transaction-feed receives attestations from Oracles, records their
+ * evidence, and when enough oracles agree, (if no risks are identified)
```

Comment:
it does not depend on whether risks are identified. https://github.com/Agoric/agoric-sdk/pull/11070/commits/23831c45757d8aa2cbc2fd3f3570b668b1162f0f